features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/rxts/api.rxt

```diff
@@ -51,7 +51,7 @@
         <field type="text">
             <name>Redirect URL</name>
         </field>
-	<field type="text">
+	    <field type="text">
```

Comment:
shall we remove these whitespaces changes

Reply:
Fixed